app/code/core/Mage/Paypal/Model/Api/Nvp.php

```diff
@@ -1164,6 +1164,7 @@ protected function _validateResponse($method, $response)
         if (isset($this->_requiredResponseParams[$method])) {
             foreach ($this->_requiredResponseParams[$method] as $param) {
                 if (!isset($response[$param])) {
+                    Mage::log("Expected PayPal field not found in NVP Response: $param");
```

Comment:
Please log more context, right now the log is not really useful. You see that sth is wrong but has no idea which order was it and which paypal method caused the issue.

Reply:
do we have any order identifying data available in this context?